backend/open_webui/retrieval/loaders/main.py

```
@@ -131,77 +132,35 @@ def load(self) -> list[Document]:
-
-            params = {"image_export_mode": "placeholder"}
-
-            if self.params:
-                if self.params.get("do_picture_description"):
-                    params["do_picture_description"] = self.params.get(
-                        "do_picture_description"
+            headers = {}
+            if self.api_key:
+                headers["X-Api-Key"] = f"Bearer {self.api_key}"
+
+            r = requests.post(
```

Ruff (0.14.8)

Probable use of `requests` call without timeout

(S113)